migrations/versions/590_remove_brief_status_column.py

```diff
@@ -0,0 +1,22 @@
+"""Remove brief status column
+
+Revision ID: 590
+Revises: 580
+Create Date: 2016-03-03 14:56:59.218753
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '590'
+down_revision = '580'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.drop_column('briefs', 'status')
+
+
+def downgrade():
+    op.add_column('briefs', sa.Column('status', sa.VARCHAR(), autoincrement=False, nullable=False))
```

Comment:
Updated with a proper downgrade. It will only set 'draft' or 'live' status, since we didn't have any closed briefs before the migration.